src/cloud/before/before-setup-env-install.md

```diff
@@ -123,7 +123,7 @@ To install Magento using the command line:
 1. Enter a CLI command with options for entering the name, email, ADMIN credentials, URL, and additional information. For a list of all options, see [Installer help commands]({{ site.baseurl }}/guides/v2.3/install-gde/install/cli/install-cli-install.html#instgde-cli-help-cmds).
 
    ```bash
-   php magento setup:install \
+   php bin/magento setup:install \
```

Comment:
```suggestion
   bin/magento setup:install \
```

We not longer need `php` at the beginning of commands. Please remove the other two as well.